src/components/FlowRunFilteredList.vue

```diff
@@ -65,23 +71,24 @@
   const selected = ref<string[]>([])
   const searchTerm = ref('')
   const searchTermDebounced = useDebouncedRef(searchTerm, 500)
+  const { value: limit } = useLocalStorage('flow-run-list-limit', 10)
```

Comment:
What's the use in having this in local storage?

Reply:
I'm just following the pattern @znicholasbrown has been using for pagination. There's a dropdown in `p-pager` that lets you select how many results to show per page. And if you change the value it remembers.